README.md

```diff
@@ -1 +1,27 @@
-# magento-cloud-patches
\ No newline at end of file
+# Magento Cloud Patches
+
+## Welcome
+Magento Cloud Patches is a set of patches, previously used within ECE-Tools to improve integration of all Magento versions with Cloud and deliver critical fixes on the fly. Starting with ece-tools `2002.0.22`/`2002.1.0`, this package is required to provide same patching flow as in ece-tools `2002.0.21` and earlier versions.
+
+## Contributing to Magento Cloud Patches Code Base
+You can submit pull requests to fix potential bugs. If you find a bug or have a new suggestion, let us know by creating a Github issue.
```

Comment:
Can they submit requests to extend functionality? (You can submit issues and pull requests to extend functionality or fix potential bugs.)

Reply:
Nope, not in this repo. I deleted this part it in this repo.